tests/aws/scenario/mythical_mysfits/test_mythical_misfits.py

```diff
@@ -0,0 +1,186 @@
+"""
+This scenario setup is based on the official AWS Modern Application Workshop sample available at
+https://github.com/aws-samples/aws-modern-application-workshop/tree/python-cdk
+
+It's originally written via TypeScript CDK but has been adapted here into a Python-based CDK application.
+"""
+
+import base64
+import json
+
+import pytest
+import requests
+
+from localstack.testing.aws.util import is_aws_cloud
+from localstack.testing.pytest import markers
+from localstack.testing.scenario.provisioning import InfraProvisioner, cleanup_s3_bucket
+from localstack.utils.strings import short_uid, to_str
+from localstack.utils.sync import retry
+from tests.aws.scenario.mythical_mysfits.stacks.mysfits_core_stack import MythicalMysfitsCoreStack
+
+STACK_NAME = "MythicalMisfitsStack"
+
+
+@pytest.mark.skipif(condition=not is_aws_cloud(), reason="not working in too many places")
+class TestMythicalMisfitsScenario:
+    """
+    Components:
+    The Mysfits microservice - Uses an Amazon Fargate container behind an NLB storing data into a DynamoDB table.
+    The Mysfits API - Provides an API with APIGateway which exposes the Mysfits microservice, as well as the Comments microservice. Uses Cognito authorizer.
+    The Comments microservice - Provides an API to update/get comments with DynamoDB, Lambda and SNS, traced with X-Ray.
+    The Users Clicks API - Pushes Click events to Kinesis Data Firehose through API Gateway to a Lambda storing enriched events in an S3 bucket via Kinesis.
+    The Recommendation API - Provides an API to give recommendations from SageMaker.
+    """
+
+    @pytest.fixture(scope="class", autouse=True)
+    def infrastructure(self, aws_client, infrastructure_setup):
+        # set up the bucket name so that we can clean it up
+        bucket_name = f"test-mythical-mysfits-{short_uid()}"
+        infra = infrastructure_setup(namespace="MythicalMisfits")
+        MythicalMysfitsCoreStack(infra.cdk_app, STACK_NAME, bucket_name=bucket_name)
+
+        infra.add_custom_teardown(lambda: cleanup_s3_bucket(aws_client.s3, bucket_name=bucket_name))
```

Comment:
this should actually not be required, as the bucket is already created within the stack. We added a feature in the provisioner that automatically cleans up buckets.

Reply:
Ah right, that's better. I think this was a leftover from when the bucket was versioned, but the utility didn't clean it up anyway. Will remove 👍